src/sigal/gallery.py

```diff
@@ -823,7 +824,7 @@ def get_albums(self, path):
             for subname, album in self.get_albums(subdir):
                 yield subname, self.albums[subdir]
 
-    def build(self, force=False):
+    def build(self, force=None):
```

Comment:
No need to change the default.

Reply:
Good catch, I was trying to avoid the implying it could just be a bool